README.md

```diff
@@ -99,6 +99,9 @@ A curated list of resources for learning about natural language processing, text
 - [A Tidytext Analysis of the Weinstein Effect](https://www.gokhanciflikli.com/post/weinstein-effect/) Dec 2013.
 - [Comparison of the Most Useful Text Processing APIs](https://www.kdnuggets.com/2018/08/comparison-most-useful-text-processing-apis.html)
 
+### Biases in NLP 
+- [Thought Leadership Blogpost-Biases in Word Embeddings] (https://neurodivergent-ai.github.io/blog/post_1.html) - Natural Language Processing News Dec 2018, Issue #35.
```

Comment:
I'm a little confused here. The post you link to is titled "what words mean" and is from a blog titled "neurodivergent ai". How do "Though Leadership Blogpost-Biases in Word Embeddings" and "Natural Language Processing News Dec 2018, Issue #35." relate? Please clarify.

Reply:
Apologies for any confusion. The blogpost was initially discovered via the NLP Newsletter that was associated with NeurIPS 2018.

However for clarity, I have proposed a different blogpost/thought leadership article discussing gender biases in datasets and word embeddings. Hopefully illustrates the topic more clearly.